docs/actions-cohortextractor.md

```diff
@@ -83,7 +81,7 @@ generate_study_cohort
   outputs:
     highly_sensitive:
       data: output/input_cohort2.csv.gz
-      ### This file is produced in compressed format. If using Stata you may need to remove the --output-format option and 
+      ### This file is produced in compressed format. If using Stata you may need to remove the --output-format option and
```

Comment:
I think this is not true anymore and they should use `.gz`?

Reply:
good catch